changelogs/fragments/k8s_facts_fix.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 bugfixes:
 - k8s_facts now returns a resources key in all situations
+- k8s_facts: fix handling of unknown resource types
```

Comment:
Sorry, this is parsing as a dict - either quote this as a string, or change the colon to a dash.

Reply:
I could've notice myself as well… Anyway, fixed again. :)